tomcat8.0/src/main/java/com/linecorp/armeria/server/http/tomcat/ArmeriaWebResourceRoot.java

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
I think we can use the similar trick with `thrift0.9`.

- Add the code only specific to Tomcat 8.0 to `tomcat8.0/src/main/java`
  - `ProtocolHandler`
    - `Tomcat85ProtocolHandler` in `tomcat`
    - `Tomcat80ProtocolHandler` in `tomcat8.0`
  - `InputBuffer` and `OutputBuffer` (we will need to make these top-level.)
    - `Tomcat85InputBuffer` in `tomcat`
    - `Tomcat85InputBuffer` in `tomcat8.0`
    - ...
- Keep most code in `tomcat` and let it detect the tomcat version and pick the classes mentioned above dynamically based on the current Tomcat version. Fail the initialization with an exception if the version mismatches.